venom-tests/samples/httpbin_blacklist.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: tyk.tyk.io/v1alpha1
```

Comment:
with your suggested modification on `config/samples/httpbin_blacklist.yaml` file, this file will be identical to [ `config/samples/httpbin_blacklist.yaml`](https://raw.githubusercontent.com/TykTechnologies/tyk-operator/6e36bcf3d213ffefaa6cc3074d8e324d66f50097/config/samples/httpbin_blacklist.yaml). Maybe, we can consider using one sample, instead of maintaining two copies of it; unless I am missing some technical details.

Reply:
I have added separate copies of all the manifests we are using so that we can modify them if needed for testing purpose.
Also will have manifests in same folder instead of jumping to different one.

Reply:
👌 makes sense. then, let's keep them